platform/darwin/src/MGLMultiPoint.mm

```diff
@@ -5,6 +5,37 @@
 #include <mbgl/util/geo.hpp>
 #include <mbgl/util/optional.hpp>
 
+bool compareByCoordinate(CLLocationCoordinate2D lhs, CLLocationCoordinate2D rhs)
+{
+    return lhs.latitude+lhs.longitude < rhs.latitude+rhs.longitude;
```

Comment:
This means coordinate{0,10} is the same as {10,0} but it's only used for sorting and as long as lhs and rhs is sorted the same way it shouldn't matter.